chart/openfaas/README.md

```diff
@@ -377,6 +377,11 @@ It is recommended that you install OpenFaaS to ARM machines [using k3sup](https:
 
 See also: [Kubernetes and Raspberry Pi in the docs](https://docs.openfaas.com/deployment/kubernetes)
 
+## Kubernetes versioning
+This Helm chart currently supports version 1.17+
```

Comment:
Is 1.17 the absolute minimum version for v1?

Reply:
I may have read the changlong incorrectly, but see https://github.com/openfaas/faas-netes/issues/709#issuecomment-727217457

Reply:
Although, I suspect that that API group isn't actually following the deprecation guidelines, since the v1 rbac group is documented in 1.15 https://v1-15.docs.kubernetes.io/docs/reference/access-authn-authz/rbac/

![image](https://user-images.githubusercontent.com/891889/99154938-f49f1580-26b3-11eb-967d-134b5c8eaf7a.png)


I will update this to 1.16+ since that is needed for the CRD update

Reply:
Sounds good to me. 1.17 may be too aggressive for some users.